src/Main.ipynb

```diff
@@ -245,6 +240,17 @@
     "            if is_human_run:\n",
     "                clear_output(wait=True)\n",
     "                display(board)\n",
+    "\n",
+    "                # Calculate cache success statistics\n",
+    "                cache_tries = player.stats[-1].get(\"cache_tries\", \"N/A\")\n",
+    "                if cache_tries == \"N/A\":\n",
+    "                    cache_hits = \"N/A\"\n",
+    "                    cache_percentage = \"N/A\"\n",
+    "                else:\n",
+    "                    cache_hits = player.stats[-1].get(\"cache_hits\", \"N/A\")\n",
+    "                    cache_percentage_value = (100 / cache_tries) * cache_hits\n",
+    "                    cache_percentage = \"{:.2f}\".format(cache_percentage_value)\n",
```

Comment:
Kann nach dem Auslagern etwas optimiert werden 😊

Reply:
Fixed in d4e37ec.